packages/library-legacy/src/connection.ts

```diff
@@ -3640,6 +3640,8 @@ export class Connection {
 
   /**
    * Returns epoch activation information for a stake account that has been delegated
+   *
+   * @deprecated Deprecated in 1.18; will be removed in a future version.
```

Comment:
nit: Maybe explicitly mention this is Solana/Agave 1.18, not Web3.js 1.18?

Reply:
Ooh. I mean… yeah. I'd have to do that for all of the comments in this file. I'll do that in the next PR.